### ISSUE TITLE
Public vs Published mismatch for --state value

### ISSUE THREAD

**Issue by DellEMCProductSecurity** [RedHatProductSecurity/cvelib#15]
Using 'public' as the value for the --state option gives this:

```
Try 'cve list -h' for help.

Error: Invalid value for '--state': 'public' is not one of 'reserved', 'published', 'rejected'
```
however changing the value to 'published' returns this:

```
ERROR: 400 Client Error: Bad Request for url: https://cveawg.mitre.org/api/cve-id?cve_id_year=2022&state=PUBLISHED
DETAILS: {'error': 'BAD_INPUT', 'message': 'Parameters were invalid', 'details': [{'msg': 'Invalid value', 'param': 'state', 'location': 'query'}]} 
```

**Comment by mprpic** [comment]
Ah, that's my mistake. The change from "public" to "published" was introduced in https://github.com/RedHatProductSecurity/cvelib/pull/10 but it wasn't released yet since it's part of CVE Services 2.x only. Later on additional changes were merged to master and those got released alongside this change, breaking it for the currently deployed prod 1.1.1 version of CVE Services.

I'll revert that change and move it to the cve-services-2.1.0 branch only so that it gets introduced only when 2.x is out.

**Comment by DellEMCProductSecurity** [comment]
Awesome, thanks for the explanation. Really appreciate the client!

**Comment by mprpic** [comment]
Fixed in 0.7.0.
